packages/components/app/styles/components/tabs.scss

```diff
@@ -6,60 +6,63 @@
 
 $indicator-height: 3px;
 
-.hds-tabs {}
+// .hds-tabs {}
 
 // Sub-components:
-// NOTE: hds-tab-list is still part of parent component may separate into child later as needed. Will consider renaming class then
-.hds-tab-list {
+// NOTE: hds-tabs__tablist-wrapper is still part of parent component, may separate into child later as needed. Will consider re-naming class then
+.hds-tabs__tablist-wrapper {
   padding-bottom: round(calc($indicator-height / 2));
   // overflow-x: auto;
   // -webkit-overflow-scrolling: touch;
   // white-space: nowrap; // move
 }
 
 // (Note: may rename later, this part of structure could still be changed as things evolve)
-.hds-tab-list__list {
-  border-bottom: 1px solid var(--token-color-border-primary);
-  list-style: none;
-  padding: 0;
-  margin: 0;
+.hds-tabs__tablist {
   display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  border-bottom: 1px solid var(--token-color-border-primary);
 }
 
 .hds-tabs__tab {
   position: relative; // for positioning temp indicator
 
-  // temporary indicator
-  /* TODO: refactor
-  &:first-child:after {
-    background: var(--token-color-foreground-action);
-    border-radius: 3px;
-    content: "";
-    display: block;
-    height: $indicator-height;
-    position: absolute;
-    left: 0;
-    right: 0;
-    bottom: round(calc($indicator-height / 2 * -1));
+  &.hds-tabs__tab--is-selected {
+    /* temporary indicator TODO: refactor WIP */
+    &::after {
```

Comment:
Note: Using a pseudo element just for now. May be re-factored once I get to implementing the animation in a later step